contact/contact.go

```diff
@@ -43,6 +48,13 @@ func New(templateFs embed.FS, emailRepo utils.EmailRepository) *Template {
 func (repository *Template) SendContactMail(c *gin.Context) {
 	var input ContactDetails
 
+	projectID := "PROJECT_ID"
+	recaptchaSiteKey := "SITE_KEY"
+	token := "TOKEN"
+	recaptchaAction := "RECAPTCHA_ACTION"
+
+	createAssessment(projectID, recaptchaSiteKey, token, recaptchaAction)
```

Comment:
No need of this assessment for now. Just get the token from contact form to this api